CLAC

```diff
@@ -0,0 +1,157 @@
+                       DEDIS / EPFL / Lausanne / CH
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+                      http://www.apache.org/licenses/
+
+   Thank you for your interest in software from DEDIS (Lab).
```

Comment:
Okay just checked the reference from the Apache Foundation and they used "Foundation" everywhere. So I guess that's where this all is coming from and then probably okay. :)